.github/workflows/centos7-ci.yml

```diff
@@ -101,6 +101,37 @@ jobs:
         docker run --rm --name consul_1 -d -p 8500:8500 consul:1.7 consul agent -server -bootstrap-expect=1 -client 0.0.0.0 -log-level info -data-dir=/consul/data
         docker run --rm --name consul_2 -d -p 8600:8500 consul:1.7 consul agent -server -bootstrap-expect=1 -client 0.0.0.0 -log-level info -data-dir=/consul/data
 
+        # start nacos server
+        nohup docker network rm nacos_net > /dev/null 2>&1 &
+        nohup docker network create nacos_net > /dev/null 2>&1 &
+        # nacos no auth server - for test no auth
+        docker run --rm -d --name nacos_no_auth --network nacos_net --hostname nacos2 --env NACOS_SERVERS="nacos1:8848 nacos2:8848" --env PREFER_HOST_MODE=hostname --env MODE=cluster --env EMBEDDED_STORAGE=embedded  --env JVM_XMS=512m --env JVM_XMX=512m --env JVM_XMN=256m -p8858:8848 nacos/nacos-server:latest
```

Comment:
Would it be better to use `nacos/nacos-server:2.0.0`?

Reply:
The nacos version should be 1.4.1,2.0.0 just released few days ago.

Reply:
@benx203
Why we can't support 2.0.0 directly? Is `2.0.0` incompatible with `1.4.1`?

Reply:
I think better way is support both 2.0.0 and 1.4.1.
Maybe support 2.0.0 later?